legend-engine-test-runner-service/src/main/java/org/finos/legend/engine/testable/service/extension/ServiceTestRunner.java

```diff
@@ -251,7 +251,10 @@ private Pair<Runtime, List<Closeable>> getTestRuntimeAndClosableResources(Runtim
                         embeddedData = connectionTestData.data;
                     }
                 }
-
+                else
```

Comment:
I don't think this is right place for this check. This would break model chain connection tests, right?
Does it not break not break any existing tests?

Reply:
yeah seems too :) 
Was looking into a bug and thought we hit a NPE regardless after this. Let me recheck that.